src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,14 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    public String getPrintableString(Printable... printables) {
+        StringBuilder builder = new StringBuilder();
```

Comment:
good! But do checkout StringJoiner class. I think the code looks cleaner with it.